### ISSUE TITLE
Standup bombs displaying odd values for current / base values in sttributes tab

### ISSUE THREAD

**Issue by blitzmann** [pyfa-org/Pyfa#1815]
![image](https://user-images.githubusercontent.com/3904767/49344068-a3212b80-f640-11e8-8176-03263bc2b68d.png)

## Bug Report

`hp` is supposed to be 4mil, but the base indicated 40. Also, `volume` seems off too. First thought iss that this is an issue with converting the values possibly?

```
[Keepstar, Keepstar fit]

[Empty Low slot]
[Empty Low slot]
[Empty Low slot]
[Empty Low slot]
[Empty Low slot]

[Empty Med slot]
[Empty Med slot]
[Empty Med slot]
[Empty Med slot]
[Empty Med slot]
[Empty Med slot]

Standup Guided Bomb Launcher II, Standup Heavy Guided Bomb
[Empty High slot]
[Empty High slot]
[Empty High slot]
[Empty High slot]
[Empty High slot]
[Empty High slot]
[Empty High slot]

[Empty Rig slot]
[Empty Rig slot]
[Empty Rig slot]

[Empty Service slot]
[Empty Service slot]
[Empty Service slot]
[Empty Service slot]
[Empty Service slot]
[Empty Service slot]
[Empty Service slot]
```

**Comment by blitzmann** [comment]
I honestly have no idea what is going on here.

In the database, the bomb (id 37849) has a value of `4000000.0` for `hp` (id 9). I can confirm that when it's set to the `original` property of modified attributes dictionary, it's still 4m.

```
        if self.__charge:
            self.__chargeModifiedAttributes.original = self.__charge.attributes
            self.__chargeModifiedAttributes.overrides = self.__charge.overrides
```

However, when this original value gets accessed in the item stats, it shows 40. 

```
   def getBaseAttrValue(self, key, default=0):
        """
        Gets base value in this order:
        Mutated value > override value > attribute value
        """
        return_value = self.itemModifiedAttributes.getOriginal(key)

        return return_value or default

```

Somehow it's being changed, and I'm not sure what / where / why. This is shown by the object location changing, when being set the HP attribute is at location `0x13e31950` whereas when it's being accessed its `0x13d9cf30`.

**Comment by DarkFenX** [comment]
Fixed by 6a2bdade8092ef7a56cc843528941b001d909f32
